package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "signalfx-collect",
-  "version": "2.0.0-beta1",
+  "version": "2.0.1-beta1",
```

Comment:
Not sure about the beta versioning but keeping it for now.

Reply:
If we want to keep it, shouldn't it be `2.0.0-beta2` then?